### PR TITLE
Add type annotation on `test_logging.py`

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

**Review by michaeloliverx — APPROVED** [review]

File: tests/middleware/test_logging.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ async def websocket_app(scope, receive, send):
                 break
 
     async def open_connection(url):
-        async with websockets.connect(url) as websocket:
+        async with websockets.client.connect(url) as websocket:
```
Comment (euri10) [inline tests/middleware/test_logging.py:107]: why this change ?

Comment (Kludex) [reply tests/middleware/test_logging.py:107]: Because there's a lazy import logic on the `websockets.__init__.py`, and that kind prevents the type checker to infer that `connect` is actually part of it... So I need to go to the submodule that actually creates it - or ignore with `# type: ignore`.
